GitUI/GitUICommands.cs

```diff
@@ -356,6 +356,44 @@ public bool StartCheckoutRemoteBranch(IWin32Window owner, string branch)
 
         #endregion
 
+        /// <summary>
+        /// Launches a new GE instance.
+        /// </summary>
+        /// <param name="arguments">The command line arguments.</param>
+        /// <param name="workingDir">The working directory for the new process.</param>
+        /// <returns>The <see cref="IProcess"/> object for controlling the launched instance.</returns>
+        public static IProcess Launch(string arguments, string workingDir = "")
```

Comment:
A better place for this general purpose functionality is probably in GitExtUtils, so plugins can leverage it as well.
But this would be a separate piece of work.